pkgs/applications/audio/radiotray-ng/default.nix

```diff
@@ -17,7 +17,9 @@
 , libxdg_basedir
 , wxGTK
 # GStreamer
+, glib-networking
```

Comment:
Unused? 

```suggestion
```

Reply:
glib-networking is an integral part of the change. the gappsWrapperArgs adds glib-networking to GIO_EXTRA_MODULES env variable so the libsoup-xxx.so can support encrypted formats.

Reply:
hmm, i guess none of the channels i tested required https support. added glib-networking to the build inputs and verified that it is now in the `GIO_EXTRA_MODULES` env var for the wrapper script. thanks for the catch!